MANIFEST.in

```diff
@@ -0,0 +1,11 @@
+include *.md
+include *.txt
+
+recursive-include docs *
+recursive-include bin *.py
+recursive-include seapy *.py
+
+include seapy/constituents.npz
+recursive-include seapy/hawaii_coast *
+recursive-include seapy/roms/cdl *
+recursive-include seapy/src *
```

Comment:
This file just tracks what goes into the source distribution.
We can automate most of that to avoid hard-coded paths,
but we need to decided what must be in the tarball first.